packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.schema.cue

```diff
@@ -51,6 +51,26 @@ DashboardV2Spec: {
   variables: [...VariableKind]
 }
 
+// Importable Dashboard
+ImportableDashboard: {
+  kind: "ImportableDashboard"
+  spec: {
+    dashboard: DashboardV2Spec
+    elements?: [...LibraryPanelExport]
+  }
+  metadata?: {
+    requirements?: [...ImportableDashboardRequirements]
+  }
+}
```

Comment:
I am not sure we need specific import / export formats / specs. 

Library panels I think should be handled in a different way. If your exporting a dashboard with library panels it should result in 1 dashboard json file and multiple files for each library element referenced. 

this "multi resource" kind feels like something we should try to avoid. Granted exporting and importing dashboards that use library panels between different instances from the UI would become more cumbersome but maybe that should be better solved with tools. or a export format that is truly generic "multi kind" format (that will work with any resource kind)